lib/compiler/src/beam_ssa_alias.erl

```diff
@@ -153,17 +154,28 @@ blk_effective_uses(#b_blk{is=Is,last=Last}) ->
 
 blk_effective_uses([I|Is], Uses0) ->
     Uses = case I of
-               #b_set{dst=Dst} ->
-                   %% The uses after the def do not count
-                   sets:del_element(Dst, Uses0);
-               _ -> % A terminator, no defs
-                   Uses0
-           end,
+                #b_set{dst=Dst} ->
+                    %% The uses after the def do not count
+                    sets:del_element(Dst, Uses0);
+                _ -> % A terminator, no defs
+                    Uses0
+            end,
```

Comment:
I see only whitespace changes on these lines. (And the indentation was correct in my Emacs before the extra whitespace was added.)

Reply:
Sloppy last minute revert of a change which did not have an impact on the performance, bad Frej! (I see that you already fixed it).